src/discord/rest_api/discord_endpoints.rs

```diff
@@ -51,11 +53,19 @@ impl ApiEndpoints {
             }
             ApiEndpoints::GetMessages(_, _, _) => client.get(self.get_url()),
         };
-
-        for (key, value) in headers {
-            request = request.header(key, HeaderValue::from_str(value.as_str()).unwrap());
+        let mut file = File::open("./public/loginInfo")?;
+        let mut token = String::new();
```

Comment:
Duplicate of get_tokens function in `src/main.rs` line 65. If you want to get a String straight away I can see adding an implantation to LoginInfo struct which will take in an enum of services and spit out the token that is associated with it, if it exists.

However this code feels redundant as you can still pass the token into it using an existing public function.